website_google_tag_manager/views/website_templates.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 ABF OSIELL <http://osiell.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <template id="layout" inherit_id="website.layout">
+        <xpath expr="//div[@id='wrapwrap']" position="before">
+            <t t-if="website and website.google_tag_manager_key">
+                <!-- Google Tag Manager (noscript) -->
+                <noscript><iframe t-attf-src="https://www.googletagmanager.com/ns.html?id=#{website.google_tag_manager_key}" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+                <!-- End Google Tag Manager (noscript) -->
+            </t>
+        </xpath>
+    </template>
+
+    <template id="web_layout" inherit_id="web.layout">
+        <xpath expr="//title" position="before">
+            <t t-if="website and website.google_tag_manager_key">
+                <!-- Google Tag Manager -->
+                <script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
+'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+})(window,document,'script','dataLayer','<t t-esc="website.google_tag_manager_key" />');</script>
```

Comment:
Why `t-esc` and not `t-field`?

Reply:
Because t-field can't be used on a t element and I need only the value, without any html element like div or span...